### ISSUE TITLE
In the Gitify menu bar (Mac) tray it has some indication that you "approved", "requested changes", or "commented" on the PR

### ISSUE THREAD

**Issue by bhingston-va** [gitify-app/gitify#682]
### 📝 Provide a description of the new feature

# Where
In the notifications tray from Gitify menu bar icon. For each tray item it would be nice to have some indication if I approved, requested changes, or just commented on it.

# Problem space
I often will approve a PR and a conversation will continue between other people (this can be very noisy). Some pull requests I want to engage and others I want to unsubscribe. However, for the life of me, I cannot recall or identify pull requests by the title and cannot remember if I reviewed it or not.

When I click on a tray item it removes it out of the tray (as expected) but then it is gone and I can't click unsubscribe through Gitify's tray item (I suppose I could through GitHubs notifications but I like Gitify).

# A suggested solution might be

Have an icon/badge for the tray item that indicates if I approved it already. Then I can choose to open or unsubscribe.

The "request-changes" comes up less frequently but is an interesting use case as it would help me prioritize which ones to review since if I'm blocking a pull request I would likely want to review those changes faster to unblock the person. Not really an issue as the person could dismiss my review.
